docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,24 @@
+= Identifying Policy in-use
+
+In Konflux, there are two places where Enterprise Contract Policies are used, integration tests and
+release pipelines.
+
+
+== Integration Tests
```

Comment:
We have this page that has a duplication of much of this information: https://konflux-ci.dev/docs/how-tos/testing/integration/editing/#configuring-the-enterprise-contract-policy

I think that it makes sense to have the actual content related to an ITS with the integration testing (i.e. policy customization). Is the information present there sufficient? We should xref to here from the customization especially if we add more information about the policy specification process (i.e. how to configure a policy that is available for referencing).

Reply:
I changed the last paragraph to link to that page. I think I still like having a little bit of information here. Let me know what you think.

Reply:
Thanks. Can we also add a link _to_ this page from the integration test pages? I think it would be helpful to direct users to how policies are evaluated within Konflux for additional context.